src/ProjectTemplates/ComponentsWebAssembly.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/Program.cs

```diff
@@ -22,6 +22,8 @@ public static async Task Main(string[] args)
             var builder = WebAssemblyHostBuilder.CreateDefault(args);
             builder.RootComponents.Add<App>("app");
 
+            builder.Logging.AddConfiguration(builder.Configuration);
```

Comment:
Do we need to do this in the project template? Can this not be done within the WebAssemblyHostBuilder?

Reply:
Good call. Done.